cel/options.go

```diff
@@ -168,7 +168,7 @@ type EvalOption int
 
 const (
 	// OptTrackState will cause the runtime to return an immutable EvalState value in the Result.
-	OptTrackState EvalOption = iota << 1
```

Comment:
I've seen in examples that people declare a `_ = iota` to force the numbers to start at 1

Reply:
Thanks, I realized I wrote this expression backwards. What I wanted was `1 << iota` rather than the other way around.